pkg/systemd/services/services.jsx

```diff
@@ -439,6 +442,14 @@ class ServicesPageBody extends React.Component {
                                     }, ex => this.setState({ error: cockpit.format(_("Listing of unit failed: $0"), ex.toString()), loadingUnits: false })));
                                 });
 
+                                let counter = 0;
+                                this.setState({ currentStatus: cockpit.format(_("Loading unit $0/$1"), counter, promisesLoad.length) });
+                                for (const p of promisesLoad) {
+                                    counter++;
+                                    p.then(() => this.setState({ currentStatus: cockpit.format(_("Loading unit $0/$1"), counter, promisesLoad.length) }))
+                                            .catch(ex => this.setState({ error: cockpit.format(_("Loading unit failed: $0"), ex.toString()), loadingUnits: false }));
```

Comment:
If a single unit fails to load, we will never see this. The next successful unit load will immediately overwrite the state again. This makes errors much harder to debug, it'd be better to log the failures (in addition/instead).

Reply:
But rendering first checks if `error` is set. So once a single unit fails, we will switch from `Loading...` to `Loading unit failed` state. Sure, it will continue to load rest of them, but the page will stay on error.

Reply:
I see, makes sense. However, could we still please log this in addition? That's much easier to see both from test logs and from user's JS console than having them to do screenshots and such.

Reply:
but indeed, this is already handled by the content of `promisesLoad`.